MacroChallengeTeam2/View/ProfileTanding.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  ProfileTanding.swift
+//  Macro Challenge Team2
+//
+//  Created by Dennis Anthony on 11/10/22.
+//
+
+import Foundation
+import SnapKit
+import UIKit
+
+final class ProfileTanding: UIView {
```

Comment:
Buat semua custom component atau view namanya belakangnya kasih "View" aja biar gampang dicari jadi ProfileTandingView

Reply:
Sama jangan lupa git convention buat commitnya, contoh:

Added UIColor Extension and also UIFont Extension.

jadi

feat: add extensions for UIColor and UIFont

Reply:
oh bener juga, okee Man noted. hari ini gua kerjain